### ISSUE TITLE
How can i deal with multiple outputs?

### ISSUE THREAD

**Issue by kwhkim** [automl/Auto-PyTorch#50]
I am wondering how I can deal with multi-headed model.

  PyTorch seems to have two distinct way of dealing multiple outputs.

  1) return y1, y2, y3

  2) return torch.cat([y1,y2,y3], axis=1)

  I think if the model output is the first, we need to modify the loss function.

**Comment by kwhkim** [comment]
It seems like auto-pytorch uses the second one.
